apps/domain/lib/domain/telemetry.ex

```diff
@@ -1,194 +1,189 @@
-    :ok
-  end
-
-  def ping do
-    PostHog.capture("ping", ping_data())
-    :ok
-  end
-
-  # How far back to count handshakes as an active device
-  # @active_device_window 86_400
-  def ping_data do
-    %{
-      allow_unprivileged_device_management: {_, allow_unprivileged_device_management},
-      allow_unprivileged_device_configuration: {_, allow_unprivileged_device_configuration},
-      local_auth_enabled: {_, local_auth_enabled},
-      logo: {_, logo}
-    } =
-      Domain.Config.fetch_source_and_configs!([
-        :allow_unprivileged_device_management,
-        :allow_unprivileged_device_configuration,
-        :local_auth_enabled,
-        :logo
-      ])
-
-    common_fields() ++
-      [
-        # devices_active_within_24h: Devices.count_active_within(@active_device_window),
-        # admin_count: Users.count_by_role(:admin),
-        # actor_count: Users.count(),
-        in_docker: in_docker?(),
-        # device_count: Devices.count(),
-        # max_devices_for_actors: Devices.count_maximum_for_a_actor(),
-        # actors_with_mfa: MFA.count_actors_with_mfa_enabled(),
-        # actors_with_mfa_totp: MFA.count_actors_with_totp_method(),
-        unprivileged_device_management: allow_unprivileged_device_management,
-        unprivileged_device_configuration: allow_unprivileged_device_configuration,
-        local_authentication: local_auth_enabled,
-        # outbound_email: Web.Mailer.active?(),
-        external_database:
-          external_database?(Map.new(Domain.Config.fetch_env!(:domain, Domain.Repo))),
-        logo_type: Domain.Config.Logo.type(logo)
-      ]
-  end
-
-  defp in_docker? do
-    File.exists?("/.dockerenv")
-  end
-
-  defp common_fields do
-    [
-      distinct_id: id(),
-      fqdn: fqdn(),
-      version: version(),
-      kernel_version: "#{os_type()} #{os_version()}"
-    ]
-  end
-
-  def id do
-    Domain.Config.fetch_env!(:domain, __MODULE__)
-    |> Keyword.fetch!(:id)
-  end
-
-  defp fqdn do
-    :web
-    |> Domain.Config.fetch_env!(Web.Endpoint)
-    |> Keyword.get(:url)
-    |> Keyword.get(:host)
-  end
-
-  defp version do
-    Application.spec(:domain, :vsn) |> to_string()
-  end
-
-  defp external_database?(repo_conf) when is_map_key(repo_conf, :hostname) do
-    is_external_db?(repo_conf.hostname)
-  end
-
-  defp external_database?(repo_conf) when is_map_key(repo_conf, :url) do
-    %{host: host} = URI.parse(repo_conf.url)
-
-    is_external_db?(host)
-  end
-
-  defp is_external_db?(host) do
-    host != "localhost" && host != "127.0.0.1"
-  end
-
-  defp os_type do
-    case :os.type() do
-      {:unix, type} ->
-        "#{type}"
-
-      _ ->
-        "other"
-    end
-  end
-
-  defp os_version do
-    case :os.version() do
-      {major, minor, patch} ->
-        "#{major}.#{minor}.#{patch}"
-
-      _ ->
-        "0.0.0"
-    end
-  end
-end
+# TODO: when app starts for migrations set env to disable connectivity checks and telemetry
```

Comment:
just FYI connectivity checks can be removed for `cloud` / 1.0. We can just rely on telemetry.